src/Http/Http.Extensions/gen/StaticRouteHandlerModel/Emitters/EndpointParameterEmitter.cs

```diff
@@ -87,15 +153,31 @@ internal static string EmitServiceParameterPreparation(this EndpointParameter en
                         {endpointParameter.EmitParameterDiagnosticComment()}
```

Comment:
(nit on existing code)

Does this need to be wrapped in a `"""` string? Why can't it just be:

```C#
builder.AppendLine(endpointParameter.EmitParameterDiagnosticComment());
```

Reply:
Indentation! 😢 

But see https://github.com/dotnet/aspnetcore/pull/46845 for some goodies.